frontend/src/component/project/Project/ProjectSettings/Settings/ArchiveProject.tsx

```diff
@@ -32,54 +30,34 @@ export const ArchiveProject = ({
     projectId,
     featureCount,
 }: IDeleteProjectProps) => {
-    const { isEnterprise } = useUiConfig();
-    const automatedActionsEnabled = useUiFlag('automatedActions');
-    const { actions } = useActions(projectId);
     const [showArchiveDialog, setShowArchiveDialog] = useState(false);
-    const actionsCount = actions.filter(({ enabled }) => enabled).length;
     const navigate = useNavigate();
+    const disabled = featureCount > 0;
+
     return (
         <StyledContainer>
             <p>
-                Before you can archive a project, you must first archive all the
-                feature flags associated with it
-                {isEnterprise() && automatedActionsEnabled
-                    ? ' and disable all actions that are in it'
-                    : ''}
-                .
+                Before you can archive a project, you must first archive all of
+                the feature flags associated with it.
             </p>
             <ConditionallyRender
                 condition={featureCount > 0}
                 show={
                     <p>
                         Currently there {featureCount <= 1 ? 'is' : 'are'}{' '}
-                        <strong>
-                            {featureCount} active feature{' '}
-                            {featureCount === 1 ? 'flag' : 'flags'}.
-                        </strong>
-                    </p>
-                }
-            />
-            <ConditionallyRender
-                condition={
-                    isEnterprise() &&
-                    automatedActionsEnabled &&
-                    actionsCount > 0
-                }
-                show={
-                    <p>
-                        Currently there {actionsCount <= 1 ? 'is' : 'are'}{' '}
-                        <strong>
-                            {actionsCount} enabled{' '}
-                            {actionsCount === 1 ? 'action' : 'actions'}.
-                        </strong>
+                        <Link component={RouterLink} to='../..'>
```

Comment:
what is the ../.. doing?

Reply:
From `project/settings` to overview, of the same project - where feature flags are

https://stackoverflow.com/a/71164807